pkg/virt-handler/isolation/detector.go

```diff
@@ -143,11 +143,9 @@ func (s *socketBasedIsolationDetector) AdjustResources(vm *v1.VirtualMachineInst
 		if err != nil {
 			return err
 		}
-		rLimit := unix.Rlimit{
-			Max: uint64(memlockSize),
-			Cur: uint64(memlockSize),
-		}
-		err = prLimit(process.Pid(), unix.RLIMIT_MEMLOCK, &rLimit)
+		memlockSizeUint64 := uint64(memlockSize)
+
```

Comment:
nit: It is not nice to place the name of the type in the name and there is no need for the newline.
You can use here `s` or `size` as it is one line away.
You can also drop this, because you can do the same in `setProcessMemoryLockRLimit`.
But frankly, why it is not returned with the correct type from `getMemlockSize`?